### PR TITLE
Update onboarding guide

### PR DESCRIPTION
**Pull request by mostafa-anm**
@codingjoe 

Would you plz have a look?

Cheers!

### CODE REVIEW

**Comment by codingjoe** [comment]
* usersnap
* hotjar
* google marketing platform

**Comment by codingjoe** [comment]
* openAI platform

**Comment by mostafa-anm** [comment]
* Add documentation about notifying team members before sometime leave to give a chance to get in touch. Rather than notifying about leaving now.

**Comment by mostafa-anm** [comment]
@codingjoe 

Added other tool and something about checking in/out.

File: accounts.md [hunk 1]
```diff
@@ -58,10 +58,56 @@ We use AWS to S3 to manage static content and utilize SES to send emails.
...
 
 We use Google Workspace at voiio. Your new Google account will give you access to most tools using SSO.
 
+
+#### Google Marketing Platform
+
+The [Google Marketing Platform][google marketing platform] includes tools like Google Analytics, Google Ads, and more, we use it for tracking feature adoptions.
+
+We have Google SSO enabled for Google Marketing Platform. Please ask your team lead to give you permissions.
+
```
Comment (codingjoe) [inline accounts.md:79]:
```suggestion
We have Google SSO enabled for Google Marketing Platform. Please ask your team lead to invite you to the correct permission groups.

```

**Review by codingjoe — APPROVED** [review]
Excellent, thanks!

**Pull request merged**